Dockerfile

```diff
@@ -1,83 +1,8 @@
-FROM node:12 as frontend-builder
 
-# Controls whether to build the frontend assets
-ARG skip_frontend_build
+FROM redash/redash:9.0.0-beta.b42121
```

Comment:
do we upgrade to beta version?

Reply:
@gfilis2 Regarding their [announcements](https://discuss.redash.io/t/redash-9-0-0-stable-release/7425) beta9 is also used in their production self hosted solution.
Most probably they will go directly into v 10.

```
The V9 beta is stable. It’s been in production on app.redash.io for more than six months. Not sure when the release will happen but the stable release will probably be exactly the same as the beta. That’s what happened with the V8 release also. 
```